src/lib/es2021.intl.d.ts

```diff
@@ -21,4 +21,106 @@ declare namespace Intl {
         formatRange(startDate: number | bigint, endDate: number | bigint): string;
         formatRangeToParts(startDate: number | bigint, endDate: number | bigint): NumberFormatPart[];
     }
-}
\ No newline at end of file
+
+    /**
+     * The locale matching algorithm to use.
+     *
+     * [MDN](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_negotiation).

```

Comment:
Not entirely certain it matters, but is the link used for the other parameter types below more useful? I suppose this one has more info about them, but the parameters page does link back to this one too.